qdax/baselines/sac.py

```diff
@@ -332,14 +436,17 @@ def _update_alpha(
         if not self._config.fix_alpha:
             # update alpha
             random_key, subkey = jax.random.split(random_key)
-            alpha_loss, alpha_gradient = jax.value_and_grad(self._alpha_loss_fn)(
+            alpha_loss, alpha_gradient = jax.value_and_grad(sac_alpha_loss_fn)(
                 training_state.alpha_params,
-                training_state.policy_params,
+                policy_fn=self._policy.apply,
+                parametric_action_distribution=self._parametric_action_distribution,
+                action_size=self._action_size,
+                policy_params=training_state.policy_params,
                 transitions=transitions,
-                random_key=subkey,
+                random_key=random_key,
```

Comment:
I think it should be `subkey` here, `random_key` is being outputted at the end of the function, so we should not use it as input of that function.

Reply:
done